src/pages/Candidates/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { Fragment, useCallback, useEffect, useState } from 'react';
+import { useHistory } from "react-router-dom";
+import { Button } from '@material-ui/core';
+
+import PageTitle from '../../layout-components/PageTitle';
+import CustomTable from '../../components/CustomTable';
+import { useEvernode } from '../../services/Evernode';
+import Loader from '../../components/Loader';
+import LabelText from '../../components/Label/LabelText'
+
+const PAGE_SIZE = 10;
+
+const Candidates = () => {
+    const history = useHistory();
+    const evernode = useEvernode();
+
+    const [candidates, setCandidates] = useState(null);
+    const [nextPageToken, setNextPageToken] = useState(null);
+    const [pageQueue, setPageQueue] = useState([]);
+    const [isCandidatesLoading, setIsCandidatesLoading] = useState(false);
+
+    const loadCandidates = useCallback(async (pageToken = null) => {
+        const data = await evernode.getCandidates(null, PAGE_SIZE, pageToken);
+        let candidateList;
+        if (data.nextPageToken) {
+            candidateList = data.data;
+            setNextPageToken(data.nextPageToken);
+        }
+        else {
+            candidateList = data;
+            setNextPageToken(null);
+        }
+
+        const tableColumns = {
```

Comment:
Shall we show the candidate type as well, Type is not in the object but can be taken from StateHelpers.getCandidateType(candidateId). And let's sort by candidate type.
```
NEW_HOOK_CANDIDATE 1
PILOTED_MODE_CANDIDATE 2
DUD_HOST_CANDIDATE 3
```
And let's show the candidate type in individual candidate view as well